providers/src/airflow/providers/jdbc/provider.yaml

```diff
@@ -25,6 +25,7 @@ state: ready
 source-date-epoch: 1728485176
 # note that those versions are maintained by release manager - do not update them manually
 versions:
+  - 4.5.3
```

Comment:
You don't / should not do it on your own - see the comment just two lines above.

The thing is that we do not know yet what will be the bump - someone else might add a new feature to the provider and it might require to bump minor rather patchlevel version - or adds a breaking change which will result in major version bump. The only exception to that is when we have a version change that is needed by another provider (say common.sql new version is needed as min-version in another provider that starts using a new feature).

Reply:
My apologies I indeed saw the comment but I was a bit confused, but indeed should only bump version when other providers are dependant on a new feature of the current one